src/org/omegat/core/machinetranslators/YandexCloudTranslate.java

```diff
@@ -162,7 +196,7 @@ protected String translate(Language sLang, Language tLang, String text) {
         String tr = extractTranslation(response);
 
         if (tr == null) {
-            return "";
+            return EMPTY_STRING;
```

Comment:
I don’t think we really need a constant for the empty string. 

Or at least give it some semantics: “no translation”, “default glossary payload” etc.